docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v2.0.0/FeatureFlag.json",
+    "type": "object",
+    "title": "An Azure App Configuration Feature Declaration",
+    "required": [
+      "id",
+      "enabled",
+      "conditions"
+    ],
+    "properties": {
+      "id": {
+        "$id": "#/properties/id",
+        "type": "string",
+        "title": "Feature ID",
+        "description": "An ID used to uniquely identify and reference the feature.",
+        "examples": [
+          "fancy-background"
+        ],
+        "pattern": "^(.*)$"
+      },
+      "description": {
+        "$id": "#/properties/description",
+        "type": "string",
+        "title": "Feature Description",
+        "description": "A description of the feature.",
+        "default": "",
+        "examples": [
+          "This will display a fancy background on the home page."
+        ],
+        "pattern": "^[a-zA-Z0-9_\\-\\.]*$"
+      },
+      "display_name": {
+        "$id": "#/properties/display_name",
+        "type": "string",
+        "title": "Feature Display Name",
+        "description": "A display name for the feature to use for display rather than the ID.",
+        "default": "",
+        "examples": [
+          "Fancy Background"
+        ],
+        "pattern": "^(.*)$"
+      },
+      "enabled": {
+        "$id": "#/properties/enabled",
+        "type": "boolean",
+        "title": "Enabled State",
+        "description": "A feature is OFF if enabled is false. If enabled is true, then the feature is ON if there are no conditions (null or empty) or if all conditions are satisfied.",
```

Comment:
Since we added the requirement_type, can we make a small update to the description "...or if ~all~ conditions are satisfied". Can you make the same update to the 1.1.0 schema?